catalyst-toolbox/src/rewards/veterans.rs

```diff
@@ -16,6 +16,15 @@ pub struct VeteranAdvisorIncentive {
     pub reputation: u64,
 }
 
+pub struct FinalRankingWithConsensus {
+    pub review_ranking: ReviewRanking,
+
+    /// This is to be used in conjunction with `ReviewRanking::is_positive()` to assess the
+    /// consensus of the boolean reply. It is either `#FO / #Rankings` or `(#Excellent + #Good) / #Rankings`.
+    /// For now we do not discriminate between Good and Excellent but this might change in the future.
+    pub negative_or_positive_consensus: Decimal,
```

Comment:
Actually I find this name quite confusing, `consensus_confidence` or something in the likes (or even `consensus` like before) is better

Reply:
ok, it was to clearly link it to the `is_positive()` because with just `consensus` there is this confusion that it's the actual consensus of the final rating while it's not (except when it's `FO`) but the documentation I added might be enough to avoid this confusion.

Reply:
> there is this confusion that it's the actual consensus of the final rating while it's not

It is in a way, with the additional difference explained in the comment. IMO naming it confidence would remove this problem since a lower score is just a general indication that not everybody agreed on the ranking without delving too much into the details

Reply:
Ok I agree. fixed by a68edef

btw, I'm not used to this GitHub review procedure: should I resolve myself each conversation like I did until now or just reply and let you resolve them (so that you can agree that it is indeed resolved)?
Do I have to re-request the review after my commit or is it not necessary?

Reply:
and [63ba8c9](https://github.com/input-output-hk/catalyst-toolbox/pull/156/commits/63ba8c9a113c960e495e4f4c43fd199e0fd3ae5d) replace the term `consensus` with `confidence` everywhere else.

Reply:
> should I resolve myself each conversation like I did until now or just reply and let you resolve them (so that you can agree that it is indeed resolved)?

It's fine as you did in these threads, I'm going to take a look anyway and add comments if I think there's more to add


> Do I have to re-request the review after my commit or is it not necessary?

Not necessary but if I forget to get back to you it's a good reminder